script/check-non-featured-docs

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# script/check-non-featured-docs: Compare the commands featured on the reference page of the site
+#                                 to the complete list of Git commands.
+#                                 Needs a checkout of the Git source at the same level as
+#                                 the checkout of the site.
+#                                 Requires Git 2.19 or later.
+#                                 Usage: GIT_VERSION=v2.23.0 SITE_VERSION=master ./script/check-non-featured-docs
+#                                 GIT_VERSION and SITE_VERSION default to HEAD
+#                                 Note: guides are also compared, but only for GIT_VERSION >= v2.18.0
+
+cd "$(dirname "$0")/.."
+
+GIT_VERSION=${GIT_VERSION:-"HEAD"}
+SITE_VERSION=${SITE_VERSION:-"HEAD"}
+
+categories=(mainporcelain \
+            ancillarymanipulators \
+            ancillaryinterrogators \
+            foreignscminterface \
+            plumbingmanipulators \
+            plumbinginterrogators \
+            synchingrepositories \
+            synchelpers \
+            purehelpers)
+
+categories_names=("Main Porcelain Commands" \
+                  "Ancillary Commands / Manipulator" \
+                  "Ancillary Commands / Interrogators" \
+                  "Interacting with Others" \
+                  "Low-level Commands / Manipulators" \
+                  "Low-level Commands / Interrogators" \
+                  "Low-level Commands / Synching Repositories" \
+                  "Low-level Commands / Synching Helpers" \
+                  "Low-level Commands / Internal Helpers")
+
+echo "Non-featured commands:"
+echo ""
+# Get featured commands
+featured_commands=$(git grep -h -o -r -E 'git[0-9a-z-]+' $SITE_VERSION app/views/shared/ref/  ':!app/views/shared/ref/_guides.html.erb' | sort)
+cd ../git/ > /dev/null
```

Comment:
Maybe use `${GIT_REPO:-../git/}` here to allow flexibility (and that variable name would match other bits, like `rake local_index`).

Just changing directory _might_ not be enough to move to another Git repo (e.g., if `$GIT_DIR` is set in the environment). But since this is just intended to help developers, I don't think we need to be too paranoid.

It might be worth checking the exit status of commands like `cd` (or just using `set -e`).

Reply:
Also, if it's just one command you're running in the sub-repo, maybe `git -C` would be easier to read than doing a `cd` and remembering to `cd -` later.

Reply:
Good points. I will change that to `git -C ...` and define `GIT_REPO`.